### ISSUE TITLE
Auto detect the different refresh ages of each environment

### ISSUE THREAD

**Issue by brendanheywood** [catalyst/moodle-local_envbar#44]
This means each env will talk to each other. We could make each non-prod env call back to prod, as the prod env is usually more visible than the non prod envs, or the other way around.

It would be better for prod to only pin on demand each non prod env, as it is away of them all, but likely not routable.

We don't ever want this to be some sort of ddos attack on prod, the update notification should be very minimal, eg once every 24 hours, but it is also pushing across a timestamp of when it was last refreshed which won't change much.

So:
* [ ] a new task, run every 24 hours
* [ ] store our last ping time to prod, if our last refresh isn't newer then don't ping
* [ ] ping a very simple rest call on the prod with a timestamp and a hash of both the url + the title. This is a very basic form of authenticating as only prod and the refreshed env's will know both the title and the url. If the title changes then this will have to be in place before the refresh gets a new date.

There is a possible edge case / opportunity - if we are refreshing from one box to a bunch of other boxes and we add another one, it might be useful if prod got notified of them all anyway. This might be a list of 'other env's' and you could one-click add it to the proper list. A poor-mans quick & simple way of getting your head around how many env's have been floating around if you are sans pipelines etc.

* [ ] So if hash doesn't match then just ignore it, but if it doesn't match AND the url is one not in our list, then go store this somewhere. Do this in a new table somewhere, or in the same table but with a flag and split the list.
